src/views/portal/AcceptedMessage.tsx

```diff
@@ -0,0 +1,167 @@
+					form on Saturday
+				</strong>
+				. Hackathon events will take place in the following rooms:
+			</p>
+			<ul style={{ listStyleType: "none", paddingLeft: 0 }}>
+				<strong>Pacific Ballroom Lobby:</strong> Check-In
+				<li>
+					<strong>Pacific Ballroom ABCD:</strong> Hacking Space
+				</li>
+				<li>
+					<strong>Woods Cove B + C:</strong> Hacking Space
+				</li>
+				<li>
+					<strong>Moss Cove A + B:</strong> Workshops and Socials
+				</li>
+			</ul>
+			<p>
+				<a href="https://map.uci.edu/?id=463#!m/255880">
+					Map to UCI Student Center
+				</a>
+				<br />
+				<a href="https://conference.studentaffairs.uci.edu/node/24">
+					Student Center Floor Map
+				</a>
+			</p>
+			<h4>On-Campus Parking</h4>
+			<p>
+				<strong>Attendees will be responsible for their own parking.</strong>{" "}
+				The closest parking structure to our event is the UCI Student Center
+				Parking Structure. This structure provides both pay-by-space ($2 per
+				hour) and permit dispensers ($1 per hour or $13 per day). You may
+				purchase a permit at any information kiosk, dispenser, or through
+				UCI&apos;s ParkbyPlate signage located in select parking areas. All
+				options accept major credit cards (Visa, MasterCard, American Express,
+				Discover). These dispensers are available from 7AM-7PM.
+			</p>
+			<p>
+				To enter the venue from the parking structure, cross Pereira Dr. and
+				enter the building with the large arch labeled “Student Center”. Once
+				you enter the building, walk straight down the hallway till the stairs.
+				Continue down the steps (there&#39;s a ramp to the left for
+				accessibility) and check in at the large desk in the lobby. If you
+				cannot get inside the venue at any point, please let an organizer know
+				via Slack, and we&#39;d be happy to help.
+			</p>
+			<p>
+				<a href="https://map.concept3d.com/?id=463#!ce/8403?ct/55161,27429,35849?m/367524?s/">
+					Map to UCI Student Center Parking Structure
+				</a>
+			</p>
+
+			<h4>General Schedule</h4>
+			<p>
+				<u>Thursday (2/2):</u>
+				<br />
+				7:00 PM - Team Formation #1 (at DBH 6011)
+			</p>
+
+			<p>
+				<u>Friday (2/3):</u> <br />
+				5:00 PM - Venue Opens for Day 1 Check-In <br />
+				6:00 PM - Dinner <br />
+				7:00 PM - Team Formation #2 <br />
+				8:00 PM - Opening Ceremony <br />
+				<strong> 9:00 PM - Hacking Begins</strong> <br />
+				12:00 AM - Venue Closes <br />
+			</p>
+
+			<p>
+				<u>Saturday (2/4):</u> <br />
+				8:00 AM - Venue Opens for Day 2 Check-In <br />
+				12:00 AM - Venue Closes <br />
+			</p>
+
+			<p>
+				<u>Sunday (2/5):</u> <br />
+				7:00 AM - Venue Opens for Day 3 Check-In <br />
+				<strong> 9:00 AM - Hacking Ends</strong> <br />
+				9:30 AM - Project Expo / Judging <br />
+				12:30 PM - Closing Ceremony <br />
+			</p>
+
+			<h4>Pre-Hack-At-UCI Checklist</h4>
+			<ol>
+				<li>Confirm your attendance by clicking on the button below</li>
```

Comment:
Need to fix wording here

Reply:
I updated it to `Confirm your attendance by clicking on the RSVP button at the bottom
					of the page`. I'm not quite sure if this is what you had in mind.

Reply:
Yeah, that's fine for now.